### ISSUE TITLE
Error occured.  `bash: make: command not found`

### ISSUE THREAD

**Issue by PalgunaGopireddy** [weiliu89/caffe#1041]
I am new to using `git` desktop version.
I installed ***git bash***. Ran following as told in `readme` file
```
git clone https://github.com/weiliu89/caffe.git
cd caffe
git checkout ssd
cp Makefile.config.example Makefile.config
```
But when I ran `make -j8`, it is showing the follwing error.
![image](https://github.com/weiliu89/caffe/assets/106730853/9e478a1f-1b2f-46b8-9b18-59b5db12b51d)

What should I do?

**Comment by tsurumi-yizhou** [comment]
You should install `make` first

**Comment by PalgunaGopireddy** [comment]
I tried installing `make` following the instructions given in `https://stackoverflow.com/a/43779544/21763726`. But the download link of `make-4.2.1-without-guile-w32-bin.zip` file is not working.

I tried installing `make` using other ways also. But they were also not prolific in resolving.

Would you suggest any way to install?

**Comment by tsurumi-yizhou** [comment]
As you're using mingw, you should add it's path into system environment variable `PATH`

**Comment by PalgunaGopireddy** [comment]
Yes, I am using `mingw` since the command prompt of Windows does not recognize the `git` command. So I installed `git-bash` from [this link](https://git-scm.com/download/win).

After that, as you told, I added the `C:\Program Files\Git\mingw64\bin` path to the environment variables. Still showing the same error `bash: make: command not found`

Have I added the wrong path?

**Comment by tsurumi-yizhou** [comment]
In Mingw the make has the name of "mingw32-make" or the similar, if it was there. In my case, mingw-make wouldn't be installed by git.

**Comment by PalgunaGopireddy** [comment]
So. Could you tell how you ran the code. I will install them. Do the same.
Have never used `caffe` it appears I need a headstart
